.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
# READ THIS!

This is a commit that upgrades bitcentive from a CanJS 3.0 to CanJS 4.0 app.  I've added comments to the __FIRST__ instance of a change required to make the application work.  If you something and there's not a comment, look through comments above and you will probably find an explanation.  If you don't find one after that, leave a question and I'll explain.  Thanks!